l20n/format/serializer.py

```diff
@@ -80,7 +80,7 @@ def dumpPattern(self, pattern):
                 str += self.dumpPlaceable(elem)
 
         if pattern['quoted']:
-            return u'"{}"'.format(str)
+            return u'"{}"'.format(str.replace('"', '\\"'))
```

Comment:
@zbraniecki will this work for placeables which contain quotes required by the syntax, such as `{ "literal string" }` or `{ NUMBER($num, style="decimal" }`? I think we should only escape inside of `TextElements` which may be part of other expressions.  I remember that when I considered this bug I thought we'd need to use a flag passed into all `dumpPlaceable` methods which would pass it on to `dumpPattern` for nested patterns and escape in line 76.